File: sequential_digits.cpp
class Solution {
public:
    vector<int> sequentialDigits(int low, int high) {
        string s = "123456789";
        vector<int> ans;

        for(int i=0; i < s.size(); i++) {
            string curr = "";
            for(int j=i; j < s.size(); j++) {
                curr += s[j];
                int num = stoi(curr);

                if(low <= num && num <= high) {
                    ans.push_back(num);
                }
            }
        }

        sort(ans.begin(), ans.end());
        return ans;
    }
};

// Time Complexity: O(1)
// Space Complexity: O(1)

File: sequential_digits.go
func sequentialDigits(low int, high int) []int {
	s := "123456789"
	ans := make([]int, 0)

	for i := 0; i < len(s); i++ {
		curr := ""
		for j := i; j < len(s); j++ {
			curr += string(s[j])
			num, _ := strconv.Atoi(curr)
            
			if low <= num && num <= high {
				ans = append(ans, num)
			}
		}
	}

	sort.Ints(ans)
	return ans
}

// Time Complexity: O(1)
// Space Complexity: O(1)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
